fake-koji/src/main/java/org/fakekoji/api/http/rest/OToolService.java

```diff
@@ -71,16 +72,42 @@ public OToolService(AccessibleSettings settings) {
 
         app.routes(() -> {
 
+            final JDKTestProjectManager jdkTestProjectManager = new JDKTestProjectManager(
+                    configManager.getJdkTestProjectStorage(),
+                    jenkinsJobUpdater
+            );
+            final JDKProjectManager jdkProjectManager = new JDKProjectManager(
+                    configManager,
+                    jenkinsJobUpdater,
+                    settings.getLocalReposRoot(),
+                    settings.getScriptsRoot()
+            );
+
             path(MISC, () -> {
                 path(REGENERATE_ALL_JOBS, () -> get(ctx -> {
-                    // TODO: implement
+                    JobUpdateResults r = new JobUpdateResults();
+                    try{
```

Comment:
please use OToolHandlerWrapper so you don't need to catch exceptions...see JDK_TEST_PROJECT endpoint below

Reply:
Thanx! Done!